palindrome.c

```diff
@@ -0,0 +1,42 @@
+#include <stdio.h>
```

Comment:
Besides that, nice job overall! You forgot to handle capitals, so words like "Hannah" should be considered palindromes but are only considered palindromes if it's all capitalized. The empty string is also valid input and should have been handled as a palindrome.